source/_layouts/default.html

```diff
@@ -8,30 +8,28 @@
         {% block head_meta %}
             <meta name="robots" content="noindex, follow">
         {% endblock %}
-        <link href="{{ site.url }}/components/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
-        <link href="{{ site.url }}/css/style.css" rel="stylesheet" type="text/css" />
+        <link rel="stylesheet" href="{{ site.url }}/build/app.css">
 
-        <link rel="apple-touch-startup-image" href="{{ site.url }}/images/jackson/2048x2048.png">
+        <link rel="apple-touch-startup-image" href="{{ site.url }}/build/jackson/2048x2048.png">
         <meta name="mobile-web-app-capable" content="yes">
         <meta name="apple-mobile-web-app-capable" content="yes">
-        <link rel="shortcut icon" sizes="76x76" href="{{ site.url }}/images/jackson/76x76.png">
-        <link rel="shortcut icon" sizes="120x120" href="{{ site.url }}/images/jackson/120x120.png">
-        <link rel="shortcut icon" sizes="128x128" href="{{ site.url }}/images/jackson/128x128.png">
-        <link rel="shortcut icon" sizes="152x152" href="{{ site.url }}/images/jackson/152x152.png">
-        <link rel="shortcut icon" sizes="196x196" href="{{ site.url }}/images/jackson/196x196.png">
-        <link rel="shortcut icon" sizes="512x512" href="{{ site.url }}/images/jackson/512x512.png">
-        <link rel="shortcut icon" sizes="1024x1024" href="{{ site.url }}/images/jackson/1024x1024.png">
-        <link rel="shortcut icon" sizes="2048x2048" href="{{ site.url }}/images/jackson/2048x2048.png">
-        <link rel="apple-touch-icon" sizes="76x76" href="{{ site.url }}/images/jackson/76x76.png">
-        <link rel="apple-touch-icon" sizes="120x120" href="{{ site.url }}/images/jackson/120x120.png">
-        <link rel="apple-touch-icon" sizes="128x128" href="{{ site.url }}/images/jackson/128x128.png">
-        <link rel="apple-touch-icon" sizes="152x152" href="{{ site.url }}/images/jackson/152x152.png">
-        <link rel="apple-touch-icon" sizes="196x196" href="{{ site.url }}/images/jackson/196x196.png">
-        <link rel="apple-touch-icon" sizes="512x512" href="{{ site.url }}/images/jackson/512x512.png">
-        <link rel="apple-touch-icon" sizes="1024x1024" href="{{ site.url }}/images/jackson/1024x1024.png">
-        <link rel="apple-touch-icon" sizes="2048x2048" href="{{ site.url }}/images/jackson/2048x2048.png">
+        <link rel="shortcut icon" sizes="76x76" href="{{ site.url }}/build/jackson/76x76.png">
```

Comment:
do we need to move the images to build, or could we keep them in the old location? this could be annoying for people who customized their templates. does encore do something with the images?

Reply:
Actually, all it does is copy them into the build/assets directory. I can move them back to how they were but personally I find it neater to have css + js + images all in the assets folder.

Reply:
Personally I prefer to have different folders for `css`, `js`, and `images` because putting everything in one tends to get messy over time. If this could be configurable that would be even better imho, giving flexibility to everyone setting up how they like it

Reply:
i see the point of having "images" that are basically icons and stuff that is referenced from the css in the same main folder as the css itself. content images is a different topic.

i don't think there is any configuration needed - encore copies them when they are in asset, otherwise the template needs to reference the original path. and having a configuration injected into the template feels too indirect to me.

what if we call that folder "icons" instead of "images" but keep it in assets? would that help to be more clear what it is meant for?

Reply:
Sorry, when I said "css + js + images all in the assets folder," I meant that there would be separate folders for css, js, and images, all within the assets folder. (Which is how the project is set up in this PR.)

Reply:
The folders that Encore copies to and from are already configured within the webpack.config.js file, see: https://github.com/sculpin/sculpin-blog-skeleton/blob/2f26a9ce05aedd5db40817e7197948ebe066da3b/webpack.config.js

Reply:
As the images in question are really only icons, I think it's a good compromise to rename the folder 'icons' in this example.

Reply:
> The folders that Encore copies to and from are already configured within the webpack.config.js file, see: [/webpack.config.js@`2f26a9c`](https://github.com/sculpin/sculpin-blog-skeleton/blob/2f26a9ce05aedd5db40817e7197948ebe066da3b/webpack.config.js)

Ah ok great! Cause if for some reason my images path changes in blog posts all paths on the site and external references would have to be updated. But looking at that I can change that to my liking and don't have to worry about it :+1: